update-embedded

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+var path = require('path');
+
+var input = '',
+    stats = { docs: 0, updates: 0 },
+    docPath,
+    doc;
+
+if (process.argv.length !== 3) {
+  console.error('Please provide the file system path to a doc.');
+  process.exit(1);
+} else {
+  docPath = process.argv[2];
+  doc = require(path.resolve(docPath));
+}
+
+var isObject = function (obj) {
```

Comment:
I'm seeing this a lot. Is it worth while considering some kind of shared library for this kind of thing? Or do you think there aren't enough examples to justify it, and / or you like each script to be independent?